corehq/apps/reports/standard/__init__.py

```diff
@@ -41,6 +41,8 @@ class CustomProjectReport(ProjectReport):
     dispatcher = CustomProjectReportDispatcher
     emailable = True
     is_public = False
+    is_support_translations = False
```

Comment:
I think this should be called `supports_translations` unless I'm misunderstanding the meaning of the variable